lib/cronex/utils.rb

```diff
@@ -32,18 +34,14 @@ def format_minutes(minute_expression)
       end
     end
 
-    def format_time(hour_expression, minute_expression, second_expression = '')
+    def format_time(hour_expression, minute_expression, second_expression = '', timezone = 'UTC')
       hour = integer(hour_expression)
-      period = hour >= 12 ? 'PM' : 'AM'
-      hour -= 12 if hour > 12
       minute = integer(minute_expression)
-      minute = format('%02d', minute)
-      second = ''
-      if present?(second_expression)
-        second = integer(second_expression)
-        second = ':' + format('%02d', second)
-      end
-      format('%s:%s%s %s', hour, minute, second, period)
+      second = second_expression == '' ? 0 : integer(second_expression)
```

Comment:
I'd go with `second_expression.to_s.empty?  ? 0 : ...`  
this will handle a `nil` value as well